observability/src/main/java/io/grpc/observability/interceptors/InetAddressUtil.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2018 The gRPC Authors
```

Comment:
Copyright should be 2022? If this was copied from another place can we look into sharing the code instead of making a copy?

Reply:
Updated. As per out discussion online, I am just leaving it here for now.